src/cmd/policy.go

```diff
@@ -84,6 +98,39 @@ func RegoFuncReadFile(ctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
 	return nil, nil
 }
 
+func RegoFuncGetGithubRepo(ctx rego.BuiltinContext, a, b *ast.Term) (*ast.Term, error) {
+
+	var org, repo string
+
+	if err := ast.As(a.Value, &org); err != nil {
+		return nil, err
+	} else if ast.As(b.Value, &repo); err != nil {
+		return nil, err
+	}
+
+	flags := runCmd.Flags()
+	githubToken, err := flags.GetString("github-token")
+	if githubToken == "" {
+		githubToken = os.Getenv("GITHUB_API_TOKEN")
+	}
+	cobra.CheckErr(err)
+
+	response, err := getClientRest().R().
+		SetHeader("Accept", "application/vnd.github+json").
+		SetHeader("Authorization", fmt.Sprintf("token %s", githubToken)).
+		Get(fmt.Sprintf("https://api.github.com/repos/%v/%v", org, repo))
+	cobra.CheckErr(err)
+
+	if response.IsError() == true {
+		log.Error().Msgf("%d: %s", response.StatusCode(), response)
+		return nil, err
+	}
+
+	reader := strings.NewReader(response.String())
```

Comment:
Yep, this is working.  Didn't need a struct after all.

Reply:
so in a rego policy you can do this?

```
github_metadata := opslevel.repo.github("org", "repo")
language := github_metadata.language
```

Reply:
sorry i didn't see your example at the top.  While that works i'm concerned it makes our UX worse then if we had a struct.

But if we have a struct then it makes it hard to get at new / non-implemented things.

I'm fine with letting us go this route for now.  But i'm fear it will cause long term policy writing pain as just do blind access into arbitrary json - IE
```
github_metadata := opslevel.repo.github("org", "repo")
language := github_metadata["language"]    <-- this part right here could fail with "key not found"
```

Reply:
That's a very good point, I figured it would be nice to be able to print what's returned in the policy and pick and choose which fields we want but I agree that once we have a pattern it will be better to have a struct defined to only pull what we need.  Would that be a pain if customers want to build custom policies and we don't have the fields they're looking for?

Reply:
Ya the struct would limit our users - but also protect them.  So thats why i'm torn.

Lets go with it for now and we can add in the struct later.